charts/helm-chart/kubernetes-dashboard/values.yaml

```diff
@@ -355,3 +355,9 @@ serviceMonitor:
   labels: {}
   ## Here annotations can be added to the serviceMonitor
   annotations: {}
+
+## Optional containers for auth addons such as keycloak-gatekeeper
```

Comment:
I'd avoid pointing to any specific addons. Let's just say auth addons here and we can merge. You'll also need to bump version.

Reply:
Thank you for the advise. Updated